environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentResourceDeletionService.java

```diff
@@ -18,20 +18,21 @@
 import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.DatalakeV4Endpoint;
 import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.StackViewV4Response;
 import com.sequenceiq.cloudbreak.auth.crn.RegionAwareInternalCrnGeneratorFactory;
+import com.sequenceiq.cloudbreak.saas.sdx.CrnAwareSdxConnector;
+import com.sequenceiq.cloudbreak.saas.sdx.TargetPlatform;
 import com.sequenceiq.distrox.api.v1.distrox.endpoint.DistroXV1Endpoint;
+import com.sequenceiq.environment.environment.domain.EnvironmentPlatform;
 import com.sequenceiq.environment.environment.domain.EnvironmentView;
 import com.sequenceiq.environment.environment.dto.EnvironmentExperienceDto;
-import com.sequenceiq.environment.experience.ExperienceConnectorService;
 import com.sequenceiq.environment.exception.EnvironmentServiceException;
-import com.sequenceiq.sdx.api.endpoint.SdxEndpoint;
-import com.sequenceiq.sdx.api.model.SdxClusterResponse;
+import com.sequenceiq.environment.experience.ExperienceConnectorService;
 
 @Service
 public class EnvironmentResourceDeletionService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(EnvironmentResourceDeletionService.class);
 
-    private final SdxEndpoint sdxEndpoint;
+    private final CrnAwareSdxConnector crnAwareSdxConnector;
```

Comment:
Wouldn't this be PlatformAwareSdxConnector?

Reply:
yes, we can name it like that too. I mean different crn type means different platform so yes, they have the same meaning in our crn world.